src/LodBody.cpp

```diff
@@ -57,8 +57,8 @@ LodBody::LodBody(std::shared_ptr<cs::core::GraphicsEngine> const& graphicsEngine
     for (auto const& s : mDEMtileSources) {
       if (s->getName() == val) {
         mPlanet.setDEMSource(s.get());
-        mGuiManager->getSideBar()->callJavascript(
-            "set_elevation_data_copyright", s->getCopyright());
+        mGuiManager->getGui()->callJavascript(
+            "CosmoScout.call", "sidebar", "setElevationDataCopyright", s->getCopyright());
```

Comment:
Also in this plugin I would prefer the `callJavascript("CosmoScout.lodBodies.setElevationDataCopyright",...)` syntax.

Reply:
Done in https://github.com/cosmoscout/csp-lod-bodies/pull/8/commits/b20752e8ea5980af209ca0ab8f0c430180627203